assets/components/stripeInlineForm/stripeInlineForm.jsx

```diff
@@ -0,0 +1,109 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import { StripeProvider, Elements, CardElement, injectStripe } from 'react-stripe-elements';
+import { getStripeKey } from 'helpers/paymentIntegrations/stripeCheckout';
+import { type Currency } from 'helpers/internationalisation/currency';
+import { type Status } from 'helpers/switch';
+import SvgArrowRightStraight from 'components/svgs/arrowRightStraight';
+import Switchable from 'components/switchable/switchable';
+import PaymentError from 'components/switchable/errorComponents/paymentError';
+
+
+// ----- Types -----//
+
+/* eslint-disable react/no-unused-prop-types */
+type PropTypes = {
+  dispatch: () => mixed,
+  stripeIsLoaded: () => mixed,
```

Comment:
Instead of passing `dispatch` down to the component, could it be handled in `mapDispatchToProps` where this component is `connect`ed?